Games/core/VegKickMeeting.js

```diff
@@ -34,6 +34,10 @@ module.exports = class VegKickMeeting extends Meeting {
     this.finished = false;
 
     for (let player of this.game.players) {
+      if (!this.members[player.id]) {
```

Comment:
we don't need this, i fixed the way kicks work by having everyone join the meeting and only enabling/ disabling kicks